api/inventorypb/agents.proto

```diff
@@ -893,6 +917,57 @@ message ChangeRDSExporterResponse {
   RDSExporter rds_exporter = 1;
 }
 
+// Add/Change ExternalExporter
+
```

Comment:
Rudimental blank line?

Reply:
And missing dot at the end

Reply:
@askomorokhov We need this blank line to not include this to swagger json.
@artemgavrilov We don't have dot at the end of other comments of this type.

Reply:
Um, what type of that comment? Is there some document that describes how properly write such comments(and other types)? Because it's not obvious and I can accidentally refactor that code fragment.

Reply:
comments like `Add/Change`, `List`, `Get` in this file which was made just to separate different models related to different endpoints.
It's not documented anywhere.

Reply:
We can't easily split this file into several due to protobuf importers, etc. Those comments just separate different sections of this huge file.